### ISSUE TITLE
Tab1 results

### ISSUE THREAD

**Issue by geek-APTX4869** [cvlab-stonybrook/Beyond-Pixels#9]
Dear author：
  How to get the results from Tab1？

**Comment by prantikhowlader** [comment]
For running on original Pascal split (1/16,1/8,1/4) ResNet 101. the hyper-parameters are:
Beta= 0.20(0-70 epochs), 0.12(70-100 epochs), after which 0.002
learning rate= 0.002
momentum=0.75
weight_decay=1e-3
scheduler= [factor=0.1, threshold=1e-3]

For running on original Pascal split (rest) ResNet 101. the hyper-parameters are:
Beta= 0.20(0-70 epochs), 0.10(70-100 epochs), after which 0.002
learning rate= 0.001
momentum=0.80
weight_decay=1e-4
scheduler= [factor=0.1, threshold=1e-4]

For running on blender Pascal splits ResNet 101. the hyper-parameters are:
Beta= 0.25(0-70 epochs), 0.15(70-100 epochs), 0.10(100-150 epochs), after which 0.001
learning rate= 0.001
momentum=0.80
weight_decay=1e-4
scheduler= [factor=0.1, threshold=1e-4]
